apps/coordinator/src/components/CreateAddress/HardwareWalletPublicKeyImporter.jsx

```diff
@@ -159,10 +159,9 @@ const HardwareWalletPublicKeyImporter = ({
             Import Public Key
           </Button>
         </Box>
-        <InteractionMessages
-          messages={interaction.messagesFor({ state: status })}
-          excludeCodes={["bip32"]}
-        />
+        {Array.isArray(InteractionMessages) && InteractionMessages.map(message => (
```

Comment:
Does this actually work to end up showing anything? I have to test it out still but looking at the code, `InteractionMessages` is a component exported from `components/InteractionMessages` which I would assume means it's never going to be an array

Reply:
I was trying to fix this and my thought process was since everything is an object in JS, I can check it via an `isArray` check. The second part of the code comes from trial and error and coincidentally it fixed the bug on my end.
I now see how this is a bad idea and makes the code a lot harder to read. Closing this PR now.